docs/topics/secured-mission-common-interaction.adoc

```diff
@@ -45,3 +115,16 @@ NOTE: This is the master realm public key from the Secured booster's {RHSSO} ser
 
 . Paste the `token` output from the client output in the _Encoded_ box.
 . You will see `Signature Verified` on the debugger page.
+
+== Using the Web Interface
+The secured endpoint also contains a web interface to interact with it. Access it using the xref:secured-endpoint[endpoint address].
+
+When using the web interface, you will go through the following steps to see how security is enforced, how authentication happens, and how the token is requested and attached to an authenticated invocation.
+
+. Open the web interface and click on _Invoke_. This sends an unauthenticated request to the Greeting service. 
+. The services responds with an `HTTP 401 Unauthenticated` status code.
+. Click _Login_ to authenticate yourself against {RHSSO}. This redirects you to the SSO server and requests your credentials. 
+. Log in as _Alice_.
+. Once authenticated you are redirected to the web interface, which includes a bearer token shown in the command line output at the bottom of the page.
+. Click _Invoke_ again to access the Greeting service.
+. Confirm there is no exception and JSON response payload displayed. This means the service accepted your bearer token and you are authorized access to the Greeting service.
```

Comment:
This should continue with:

- logout
- try invoking again
- login as admin/admin
- try invoking again -- will see different error because the admin user, while authenticated, isn't authorized